community templates/Sensitive Data Discovery with Delphix/manifest.json

```diff
@@ -1,38 +1 @@
in=\"flood\" in2=\"blur\" operator=\"in\" result=\"comp\"/><feOffset in=\"comp\" result=\"offsetBlur\" dx=\"0\" dy=\"0\"/><feMerge><feMergeNode in=\"offsetBlur\"/><feMergeNode in=\"SourceGraphic\"/></feMerge></filter><path d=\"M 0,0 L 12,0 L 12,12 L 0,12 z\" fill=\"#ffffff\" stroke=\"none\" transform=\"matrix(1, 0, 0, 1, 0, 0)\"/></g><path d=\"M 9.38802,1.4220000000000002 L 7.9660199999999985,0 L 4.693999999999999,3.272 L 1.4220000000000002,0 L 0,1.4220000000000002 L 3.272,4.694000000000001 L 0,7.96597 L 1.4220000000000002,9.387970000000001 L 4.693999999999999,6.116 L 7.9660199999999985,9.387970000000001 L 9.38802,7.96597 L 6.116,4.694000000000001 L 9.38802,1.4220000000000002 z\" fill=\"#A80000\" stroke=\"none\" transform=\"matrix(1, 0, 0, 1, 1.3059900000000004, 1.3060149999999995)\"/></g></g><g transform=\"matrix(1, 0, 0, 1, 0, 54)\"><g transform=\"matrix(1, 0, 0, 1, 0, 6)\"/></g></g></g></g><g transform=\"matrix(1, 0, 0, 1, 1921, 400)\"><g transform=\"matrix(1, 0, 0, 1, 1, 0)\"><g transform=\"matrix(1, 0, 0, 1, 0, 0)\"><path d=\"M 2,0 L 189,0 C 189.89543050033842,0 191,1.1045694996615871 191,2 L 191,91.80913899932317 C 191,92.70456949966157 189.89543050033842,93.80913899932317 189,93.80913899932317 L 2,93.80913899932317 C 1.1045694996615871,93.80913899932317 0,92.70456949966157 0,91.80913899932317 L 0,2 C 0,1.1045694996615871 1.1045694996615871,0 2,0 z\" fill=\"none\" stroke=\"#C8C8C8\" stroke-width=\"1\" stroke-linecap=\"butt\" stroke-linejoin=\"miter\" stroke-miterlimit=\"10\" transform=\"matrix(1, 0, 0, 1, 0.5, 0.5)\"/><g transform=\"matrix(1, 0, 0, 1, 0, 0)\" clip-path=\"url(#CLIP3971002)\"><g transform=\"matrix(1, 0, 0, 1, -0.7999999999999894, 1.60456949966158)\"><g transform=\"matrix(1, 0, 0, 1, 0.8, 0.8)\"><g transform=\"matrix(1, 0, 0, 1, 0, 0)\"><rect x=\"0\" y=\"0\" width=\"192\" height=\"90\" fill=\"#F4F4F4\" stroke=\"none\" transform=\"matrix(1, 0, 0, 1, 0, 0)\"/><g transform=\"matrix(1, 0, 0, 1, 0, 0)\"><g transform=\"matrix(1, 0, 0, 1, 0, 0)\"><rect x=\"0\" y=\"0\" width=\"192\" height=\"25\" fill=\"#DCDCDC\" stroke=\"none\" transform=\"matrix(1, 0, 0, 1, 0, 0)\"/><g transform=\"matrix(1, 0, 0, 1, 0, 0)\" clip-path=\"url(#CLIP3971004)\"><text x=\"0\" y=\"10.506005859375001\" style=\"font: 12px Segoe UI\" text-anchor=\"start\" fill=\"#515151\" stroke=\"none\" transform=\"matrix(1, 0, 0, 1, 5, 6.5)\">Get Metadata</text></g><g transform=\"matrix(1, 0, 0, 1, 0, 0)\" clip-path=\"url(#CLIP3971006)\"><g transform=\"matrix(1, 0, 0, 1, 142, -3)\"><path d=\"M 1.8409491661026458,0 L 3.6818983322052947,0 C 4.506119348655733,0 5.52284749830794,1.0167281496522078 5.52284749830794,1.8409491661026458 L 5.52284749830794,3.681898332205292 C 5.52284749830794,4.50611934865573 4.506119348655733,5.522847498307938 3.6818983322052947,5.522847498307938 L 1.8409491661026458,5.522847498307938 C 1.0167281496522078,5.522847498307938 0,4.50611934865573 0,3.681898332205292 L 0,1.8409491661026458 C 0,1.0167281496522078 1.0167281496522078,0 1.8409491661026458,0 z
```

Comment:
Please add formatting for this file as well

Reply:
done